### DISCUSSION TITLE
Is there an option to disable paredit without disabling all the keybinding?

### DISCUSSION THREAD

**Discussion by ahmedlhanafy** [BetterThanTomorrow/calva#967] [Q&A]
I recently started fiddling around with Calva and I'm really enjoying it, however, I don't really like Paredit and I want to disable it. When I disabled it through "Toggle keybindings", evaluating forms with keyboard shortcut stopped working. Was wondering if there is a way to disable Paredit while preserving the eval shortcuts.

**Comment by bpringe** [comment]
![image](https://user-images.githubusercontent.com/12722744/105113857-9e1ad280-5a7a-11eb-9d68-9f3e42b11d0f.png)

Does setting it to `none` in settings do what you want?

**Comment by bpringe** [comment]
May I ask what things about paredit make you not like it? We are in the process of changing a few default keybindings that we got multiple complaints about over time due to conflicts with VS Code (or typical elsewhere) defaults. Namely, some people are used to using `ctrl+leftArrow/rightArrow` for navigation, and `ctrl+w` for closing tabs.

We've altered these to be less disruptive in those regards in #956, which will be merged soon.

You can also remap any individual shortcuts you don't like, but to each their own, so if disabling floats your boat, please do that.

**Comment by PEZ** [comment]
Paredit should stay out of your way if you just click the icon in the status bar with the lambda icon and square brackets. It should toggle to round brackets and then you're on your own with the structure of the code. I would advice to keep strict mode on and just get used to using `alt+backspace`/`alt+delete` when some bracket really needs to be removed.
